doc/sailfish-webview-webenginesettings.qdoc

```diff
@@ -0,0 +1,180 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*!
+    \qmltype WebEngineSettings
+    \inherits Item
+    \inqmlmodule Sailfish.WebEngine
+    \ingroup sailfishos-webview
+    \brief Provides access to the global Web engine settigns.
+
+    The \c{WebEngineSettings} component is a singleton which provides access to
+    the global Web engine settings.
+
+    The \c{WebEngineSettings} QML type is exposed through the
+    \c{Sailfish.WebEngine} QML import namespace.
+
+    Exaple of use:
+
+    \qml
+        import QtQuick 2.0
+        import Sailfish.Silica 1.0
+        import Sailfish.WebView 1.0
+        import Sailfish.WebEngine 1.0
+
+        ApplicationWindow {
+            id: root
+            initialPage: Component {
+                WebViewPage {
+                    id: page
+                    WebView {
+                        anchors.fill: parent
+                        active: true
+                        url: "http://www.sailfishos.org"
+                    }
+
+                    Component.onCompleted: {
+                        WebEngineSettings.autoLoadImages = false
+                        WebEngineSettings.javascriptEnabled = false
+                    }
+                }
+            }
+        }
+    \endqml
+
+    \sa WebView WebEngine
+*/
+
+/*!
+    \qmlproperty bool WebEngineSettings::initialized
+    \brief Whether the engine settings have been initialized.
+
+    This property returns true when the engine settings have been initialized
+    and false otherwise
+
+    \sa WebView::initialize WebEngine::initialize
+*/
+
+/*!
+    \qmlproperty bool WebEngineSettings::autoLoadImages
+    \brief Whether to automatically load images.
+
+    When set to \c true images will be automatically downloaded and rendered.
+    When set to \c false images will not be downloaded.
+
+    The default value is \c true.
+
+    This corresponds to the "permissions.default.image" gecko preference.
+
+    Setting this property to \c true can help avoid tracking by external
+    servers when rendering local content, but may prevent a page from rendering
+    correctly.
+*/
+
+/*!
```

Comment:
why is the documentation duplicated for QMozEngineSettings and WebEngineSettings?  Do we need to provide C++ documentation?

Reply:
I'm not sure whether it applies to the WebEngineSettings, but at least for the WebEngine there seemed to be some important differences between the two (related to initialisation).

Nonetheless, I'm very happy to include/remove whatever is appropriate: open to instruction on this.